### ISSUE TITLE
Design Guidelines

### ISSUE THREAD

**Issue by ishandeveloper** [CheshtaK/bookface#12]
**Is your feature request related to a problem? Please describe.**
The current User Interface of the app is very cluttered, and just not that pleasing for the user and I believe that before we start working on other sections of the app, we should address this issue,

**Describe the solution you'd like**
I can generate several mockups as per the requirement of the app, which strictly follow a minimalistic and clean look. As an add-on feature we can also add support for both dark-mode and light mode, which can be implemented a Provider based listener, throughout the app.

**Additional context**
Let me know your views on the subject, and If you'd agree, I can start working on the same.

**Comment by CheshtaK** [comment]
Yes @ishandeveloper! That's a valid issue. Let me know when you're done with #9 . I'll assign you this.

**Comment by ishandeveloper** [comment]
Hey @CheshtaK can you please assign this issue to me now?

**Comment by ishandeveloper** [comment]
I have put some of my thoughts into these mockups, most of the important sections are done and I think we can start working on these sections for now and see how it turns out. Later on, we can create mockups for other sections as well.

I have attached screenshots of the mock-ups below. Kindly have a look at them and let me know your views.

Once you approve these designs, I'll write up a `design.md` file containing all the design guidelines for the project.
<details>
<summary> Splash Screen </summary>
<img src="https://user-images.githubusercontent.com/54989142/85202613-3c985700-b325-11ea-9172-c37ae3f3326b.png" height="500em"/>
</details>
<details>
<summary> Login Page</summary>
<img src="https://user-images.githubusercontent.com/54989142/85202581-fb07ac00-b324-11ea-899e-6eebd789b985.png" height="500em">
</details>
<details>
<summary> Categories</summary>
<img src="https://user-images.githubusercontent.com/54989142/85202651-723d4000-b325-11ea-83f0-47e7bf8c7fe9.png" height="500em">
</details>
<details>
<summary> Feeds/PostsSection</summary>
<img src="https://user-images.githubusercontent.com/54989142/85202660-913bd200-b325-11ea-8240-5e8a9c14a2d3.png" height="500em">
</details>
<details>
<summary> Chat / Messages</summary>
<img src="https://user-images.githubusercontent.com/54989142/85202671-ad3f7380-b325-11ea-8f15-9871e96a0538.png" height="500em">
</details>

**Comment by CheshtaK** [comment]
These look great @ishandeveloper ! You can open up a PR. Let's leave the Chat/Messages page for now. Can you open a new issue for the same? We'll work on that later.

**Comment by ishandeveloper** [comment]
@CheshtaK I have linked a PR. Please have a look at the same.
